### PR TITLE
RNA-seq tools added in readme

### PR DESCRIPTION
**Pull request by pavanvidem**
Added RNA-seq tools and created new categories with NGS prefix

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -222,6 +246,7 @@ In this section we list all tools that have been integrated in the RNA workbench
  - [Sebastian Will](https://github.com/s-will)
  - [Markus Wolfien](https://github.com/mwolfien)
  - [Dilmurat Yusuf](https://github.com/dyusuf)
+ - [Pavankumar Videm] (https://github.com/pavanvidem)
```
Comment (bgruening) [inline README.md]: The space is too much.

File: README.md [hunk 2]
```diff
@@ -190,18 +190,42 @@ In this section we list all tools that have been integrated in the RNA workbench
...
 <p align="right"><a href="#top">&#x25B2; back to top</a></p>
 
+## NGS | Quantification
+| Tool | Description | Reference |
+| ---- | ----------- | --------- |
+| [featurecounts](http://bioinf.wehi.edu.au/featureCounts/) | a ultrafast and accurate read summarization program | [Liao et al. 2014](http://dx.doi.org/10.1093/bioinformatics/btt656) |
+| [htseq-count](http://www-huber.embl.de/HTSeq/doc/count.html) | Tool for counting reads in features | [Anders et al. 2015](https://dx.doi.org/10.1093%2Fbioinformatics%2Fbtu638) |
+| [Sailfish](http://www.cs.cmu.edu/~ckingsf/software/sailfish/) | Rapid Alignment-free Quantification of Isoform Abundance | [Patro et al. 2014](http://dx.doi.org/10.1038/nbt.2862) |
+| [Salmon](https://combine-lab.github.io/salmon/) | Fast, accurate and bias-aware transcript quantification | [Patro et al. 2017](http://dx.doi.org/10.1038/nmeth.4197) |
+## NGS | Transcript Assembly
```
Comment (bgruening) [inline README.md]: The `back to top` links are missing.

**Comment by bgruening** [comment]
Thanks @pavanvidem!

File: README.md (hunk 1 above)
Comment (bebatut) [inline README.md]: I planned to fix it

File: README.md (hunk 2 above)
Comment (bebatut) [inline README.md]: I will fix it, I will work a bit on the form :)

**Pull request merged**